src/webui/www/private/scripts/prop-trackers.js

```diff
@@ -80,42 +105,48 @@ window.qBittorrent.PropTrackers ??= (() => {
-                                    break;
-                                case 2:
-                                    status = "QBT_TR(Working)QBT_TR[CONTEXT=TrackerListWidget]";
-                                    break;
-                                case 4:
-                                    status = "QBT_TR(Not working)QBT_TR[CONTEXT=TrackerListWidget]";
-                                    break;
-                            }
-                        }
-
                         const row = {
                             rowId: tracker.url,
                             tier: (tracker.tier >= 0) ? tracker.tier : "",
+                            bt_version: "",
                             url: tracker.url,
-                            status: status,
+                            status: trackerStatusText(tracker),
                             peers: (tracker.num_peers >= 0) ? tracker.num_peers : "QBT_TR(N/A)QBT_TR[CONTEXT=TrackerListWidget]",
                             seeds: (tracker.num_seeds >= 0) ? tracker.num_seeds : "QBT_TR(N/A)QBT_TR[CONTEXT=TrackerListWidget]",
                             leeches: (tracker.num_leeches >= 0) ? tracker.num_leeches : "QBT_TR(N/A)QBT_TR[CONTEXT=TrackerListWidget]",
                             downloaded: (tracker.num_downloaded >= 0) ? tracker.num_downloaded : "QBT_TR(N/A)QBT_TR[CONTEXT=TrackerListWidget]",
                             message: tracker.msg,
+                            next_announce: tracker.next_announce,
+                            min_announce: tracker.min_announce,
+                            _isTracker: true,
+                            _hasEndpoints: tracker.endpoints && (tracker.endpoints.length > 0),
                             _sortable: !tracker.url.startsWith("** [")
                         };
 
                         torrentTrackersTable.updateRowData(row);
+
+                        if (tracker.endpoints !== undefined) {
+                            for (const endpoint of tracker.endpoints) {
+                                const row = {
+                                    rowId: `endpoint|${tracker.url}|${endpoint.name}`,
+                                    tier: "",
+                                    bt_version: `v${endpoint.bt_version}`,
+                                    url: endpoint.name,
+                                    status: trackerStatusText(endpoint),
+                                    peers: (endpoint.num_peers >= 0) ? endpoint.num_peers : "QBT_TR(N/A)QBT_TR[CONTEXT=TrackerListWidget]",
+                                    seeds: (endpoint.num_seeds >= 0) ? endpoint.num_seeds : "QBT_TR(N/A)QBT_TR[CONTEXT=TrackerListWidget]",
+                                    leeches: (endpoint.num_leeches >= 0) ? endpoint.num_leeches : "QBT_TR(N/A)QBT_TR[CONTEXT=TrackerListWidget]",
+                                    downloaded: (endpoint.num_downloaded >= 0) ? endpoint.num_downloaded : "QBT_TR(N/A)QBT_TR[CONTEXT=TrackerListWidget]",
```

Comment:
You can extract the `N/A` string to a local variable so the translators won't see duplicate strings.